cdc/sinkv2/eventsink/event.go

```diff
@@ -32,7 +32,7 @@ type CallbackFunc func()
 type CallbackableEvent[E TableEvent] struct {
 	Event       E
 	Callback    CallbackFunc
-	TableStatus *pipeline.TableState
+	TableStatus pipeline.TableState
```

Comment:
Why need `TableStatus` and why change to value?

Reply:
It's a mistake. Fixed and added test for it.